src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static Random random;
+
+    public ColorSupplier(Random random) {
+        this.random = random;
+    }
```

Comment:
better to initialize this in 6 line

Reply:
What did you mean? I try change it and did work/